docs/confounding-variables.md

```diff
@@ -0,0 +1,40 @@
+# Confounding Variables
+
+A confounding variable is a factor that influences independent variable and dependant variable. 
+A confounding variable can mix with the effect of an independent variable on the dependent variable thus leading to erronous conclusions about the experiment.
+
+At any time during your experiment, the data manager allows you to capture confounding variables. 
```

Comment:
what do you mean with any time of the experiment? This is done in the lab, not in the web service.

Reply:
During the lifetime of the experiment. So from experiment creation on already. You do not need to have samples registered or any other experiment setup (exp. variables, groups)

Reply:
I am not sure if this is a community-accepted and used term: experiment life-time. An experiment is planned, conducted and its result analyzed. 

Do you mean at any step of the data management process in Data Manager?